### PR TITLE
Spin to win.

### PR DESCRIPTION
**Pull request by Yann21**
I like the spinning animation but I still have to find a way to remove the keyboard. For some other day.

Also I kinda changed some ID naming convention.
1. For edittext_profile I believe it's more readable if we put it like this editText_profile, and I saw it on some forums
2. I changed login_button to button_login
3. If there's something that I changed and you don't like tell me and I'll bring it back. I kinda jumped on the naming because I'm used to coding on my own and it was a good way of diving into unknown sections of code. Don't be upset :stuck_out_tongue: .

Anyway more pull requests will be coming in. Enjoy your week-end.

### CODE REVIEW

**Comment by Yann21** [comment]
test

File: app/src/main/java/com/example/dressme/util/FirebaseWrapper.kt [hunk 1]
```diff
@@ -98,7 +106,7 @@ class FirebaseWrapper {
 data class Item(val name: String            =  "",
                 val desc_text: String       =  "",
                 val item_image_uri: String  ?= null,
-                val owner_user: OwnerUser   ?= null)
+                val owner_user: OwnerUser?= null)
```
Comment (bboyifeel) [inline app/src/main/java/com/example/dressme/util/FirebaseWrapper.kt]: why did you remove tab?? bring it back

Comment (Yann21) [reply app/src/main/java/com/example/dressme/util/FirebaseWrapper.kt]: Not made on purpose, this was a mistake
